<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserForm</class>
 <widget class="QDialog" name="AddUserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить сотрудника</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_Email"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_Otch"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_Birthdays">
       <property name="text">
        <string>Дата рождения</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_Otch">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_Post"/>
     </item>
     <item row="3" column="2">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_Email">
       <property name="text">
        <string>Почта</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Post">
       <property name="text">
        <string>Должность</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editline_Name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="editline_Famili"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_Name">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Famili">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editline_Famili</tabstop>
  <tabstop>editline_Name</tabstop>
  <tabstop>lineEdit_Otch</tabstop>
  <tabstop>lineEdit_Post</tabstop>
  <tabstop>lineEdit_Email</tabstop>
  <tabstop>dateEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddUserForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddUserForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
